### ISSUE TITLE
when i run intel_extension_for_pytorch.quantization import prepare  prepare(user_model.eval(), qconfig, example_inputs=example_inputs) killed

### ISSUE THREAD

**Issue by duyanyao** [intel/intel-extension-for-pytorch#532]
### Describe the issue

Now I'm replicating this [implementation，](https://intel.github.io/intel-extension-for-pytorch/llm/cpu/#compile-from-source)
pytorch=2.1.0.dev20230711+cpu
intel-extension-for-pytorch=2.1.0.dev0+cpu.llm
but an error occurred while executing the step
## Llama 2 quantization
python run_llama_int8.py --ipex-smooth-quant --lambada --output-dir "saved_results" --jit --int8-bf16-mixed -m <LLAMA MODEL_ID>
Directly show killed.
Please help to see what causes it

**Comment by hueyline** [comment]
Hi @duyanyao, I will try to reproduce the issue and get back to you.

**Comment by duyanyao** [comment]
There is another problem, when running the script, the parameter connector passed is -, but the code uses _ for the branch, so even if the parameter is added, it cannot be called to the branch
I changed all the connectors in the code to _, which caused the above error

**Comment by duyanyao** [comment]
The current error is as follows
![Uploading ipex_error.png…]()

**Comment by duyanyao** [comment]
**pip list**
accelerate                  0.27.2
aiohttp                     3.9.3
aiosignal                   1.3.1
annotated-types             0.6.0
async-timeout               4.0.3
attrs                       23.2.0
certifi                     2024.2.2
charset-normalizer          3.3.2
cmake                       3.28.3
contextlib2                 21.6.0
contourpy                   1.2.0
cpuid                       0.0.11
cpuid-native                0.0.8
cycler                      0.12.1
datasets                    2.17.0
deepspeed                   0.11.2+94873fe1
Deprecated                  1.2.14
dill                        0.3.8
exceptiongroup              1.2.0
filelock                    3.13.1
fonttools                   4.49.0
frozenlist                  1.4.1
fsspec                      2023.10.0
hjson                       3.1.0
huggingface-hub             0.20.3
hypothesis                  6.98.6
idna                        3.6
importlib-resources         6.1.1
intel-extension-for-pytorch 2.1.0.dev0+cpu.llm
Jinja2                      3.1.3
joblib                      1.3.2
kiwisolver                  1.4.5
MarkupSafe                  2.1.5
matplotlib                  3.8.3
mkl-include                 2024.0.0
mkl-static                  2024.0.0
mpmath                      1.3.0
multidict                   6.0.5
multiprocess                0.70.16
networkx                    3.2.1
neural-compressor           2.2
ninja                       1.11.1.1
numpy                       1.26.4
oneccl-bind-pt              2.1.0+cpu
opencv-python               4.9.0.80
packaging                   23.2
pandas                      2.2.0
pillow                      10.2.0
pip                         23.3.1
prettytable                 3.9.0
protobuf                    3.20.3
psutil                      5.9.8
py-cpuinfo                  9.0.0
pyarrow                     15.0.0
pyarrow-hotfix              0.6
pycocotools                 2.0.7
pydantic                    2.6.1
pydantic_core               2.16.2
pynvml                      11.5.0
pyparsing                   3.1.1
python-dateutil             2.8.2
pytz                        2024.1
PyYAML                      6.0.1
regex                       2023.12.25
requests                    2.31.0
safetensors                 0.4.2
schema                      0.7.5
scikit-learn                1.4.1.post1
scipy                       1.12.0
sentencepiece               0.1.99
setuptools                  68.2.2
six                         1.16.0
sortedcontainers            2.4.0
sympy                       1.12
threadpoolctl               3.3.0
tokenizers                  0.13.3
torch                       2.1.0.dev20230711+cpu
torchaudio                  2.1.0.dev20230711+cpu
torchvision                 0.16.0.dev20230711+cpu
tqdm                        4.66.2
transformers                4.28.1
typing_extensions           4.9.0
tzdata                      2024.1
urllib3                     2.2.0
wcwidth                     0.2.13
wheel                       0.41.2
wrapt                       1.16.0
xxhash                      3.4.1
yarl                        1.9.4
zipp                        3.17.0

I used quantization scripts from other branches(--branch llm_feature_branch)，Now the error is as follows：

/root/miniconda3/envs/py39/lib/python3.9/site-packages/intel_extension_for_pytorch/quantization/_quantize.py:94: UserWarning: BatchNorm folding failed during the prepare process.
  warnings.warn("BatchNorm folding failed during the prepare process.")
/root/miniconda3/envs/py39/lib/python3.9/site-packages/intel_extension_for_pytorch/cpu/transformers/attentions.py:80: UserWarning: To copy construct from a tensor, it is recommended to use sourceTensor.clone().detach() or sourceTensor.clone().detach().requires_grad_(True), rather than torch.tensor(sourceTensor).
  else torch.tensor(expanded_attn_mask)
/root/miniconda3/envs/py39/lib/python3.9/site-packages/intel_extension_for_pytorch/cpu/transformers/attentions.py:81: UserWarning: To copy construct from a tensor, it is recommended to use sourceTensor.clone().detach() or sourceTensor.clone().detach().requires_grad_(True), rather than torch.tensor(sourceTensor).
  + torch.tensor(combined_attention_mask)
/root/miniconda3/envs/py39/lib/python3.9/site-packages/intel_extension_for_pytorch/cpu/transformers/attentions.py:428: UserWarning: To copy construct from a tensor, it is recommended to use sourceTensor.clone().detach() or sourceTensor.clone().detach().requires_grad_(True), rather than torch.tensor(sourceTensor).
  torch.tensor(past_key_value[3] + query.shape[1], dtype=torch.long),
/root/miniconda3/envs/py39/lib/python3.9/site-packages/intel_extension_for_pytorch/cpu/transformers/attentions.py:20: TracerWarning: torch.tensor results are registered as constants in the trace. You can safely ignore this warning if you use this function to create tensors out of constant variables that would be the same every time you call this function. In any other case, this might cause the trace to be incorrect.
  torch.tensor(torch.finfo(dtype).min, device=device),
/root/miniconda3/envs/py39/lib/python3.9/site-packages/intel_extension_for_pytorch/cpu/transformers/attentions.py:80: TracerWarning: torch.tensor results are registered as constants in the trace. You can safely ignore this warning if you use this function to create tensors out of constant variables that would be the same every time you call this function. In any other case, this might cause the trace to be incorrect.
  else torch.tensor(expanded_attn_mask)
/root/miniconda3/envs/py39/lib/python3.9/site-packages/intel_extension_for_pytorch/cpu/transformers/attentions.py:81: TracerWarning: torch.tensor results are registered as constants in the trace. You can safely ignore this warning if you use this function to create tensors out of constant variables that would be the same every time you call this function. In any other case, this might cause the trace to be incorrect.
  + torch.tensor(combined_attention_mask)
/root/miniconda3/envs/py39/lib/python3.9/site-packages/intel_extension_for_pytorch/quantization/_quantization_state_utils.py:442: TracerWarning: Converting a tensor to a Python number might cause the trace to be incorrect. We can't record the data flow of Python values, so this value will be treated as a constant in the future. This means that the trace might not generalize to other inputs!
  args, scale.item(), zp.item(), dtype
/root/miniconda3/envs/py39/lib/python3.9/site-packages/intel_extension_for_pytorch/quantization/_quantization_state.py:452: TracerWarning: Converting a tensor to a Python boolean might cause the trace to be incorrect. We can't record the data flow of Python values, so this value will be treated as a constant in the future. This means that the trace might not generalize to other inputs!
  if scale.numel() > 1:
/root/miniconda3/envs/py39/lib/python3.9/site-packages/intel_extension_for_pytorch/cpu/transformers/attentions.py:428: TracerWarning: torch.tensor results are registered as constants in the trace. You can safely ignore this warning if you use this function to create tensors out of constant variables that would be the same every time you call this function. In any other case, this might cause the trace to be incorrect.
  torch.tensor(past_key_value[3] + query.shape[1], dtype=torch.long),
Killed

The final result was still killed

**Comment by hueyline** [comment]
Hi @duyanyao, I can run the Llama 2 quantization and int8 performance test without having any issue. Please see attached. Thus, can you share the error you encountered from v2.1.0.dev+cpu.llm branch and the steps you used to perform the model conversion?

![image](https://github.com/intel/intel-extension-for-pytorch/assets/97221497/925e00fb-e04b-4716-b706-bfbcd3696e71)
![image](https://github.com/intel/intel-extension-for-pytorch/assets/97221497/1c822ebc-572e-4206-8dde-3842851e9173)
